code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -85,4 +105,18 @@
 			visible_message(message)
 		else
 			audible_message(message)
-	return
\ No newline at end of file
+	return
+
+/mob/living/carbon/alien/humanoid/proc/change_icon(icon_name)
+	var/original_icon = icon
+	var/original_icon_state = icon_state
+
+	//change the icon for fun
+	icon = 'icons/mob/alien_emotes.dmi'
+	icon_state = "[icon_name]"
+
+	//change the icon back after a while as fun is a precious resource
+	spawn(emote_length)

```

Comment:
... our lord in heaven, your will be done, on earth as it is in heaven. That's the jingle, right?